src/app/embed/adapters/merge.adapter.ts

```diff
@@ -1,18 +1,25 @@
-import {Injectable} from '@angular/core';
-import { Observable } from 'rxjs/Observable';
-import { QSDAdapter } from './qsd.adapter';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { DraperAdapter } from './draper.adapter';
 import { FeedAdapter } from './feed.adapter';
-import { AdapterProperties, hasMinimumParams, DataAdapter, getMergedValues } from './adapter.properties';
+import { QSDAdapter } from './qsd.adapter';
+import { AdapterProperties, PropNames, DataAdapter } from './adapter.properties';
 
 const NO_EMIT_YET = Symbol();
 
 @Injectable()
 export class MergeAdapter {
 
+  static REQUIRED = ['audioUrl', 'title', 'subtitle', 'subscribeUrl', 'feedArtworkUrl'];
+
   private adapters: DataAdapter[];
 
-  constructor(private feedAdapter: FeedAdapter, private qsdAdapter: QSDAdapter) {
-    this.adapters = [qsdAdapter, feedAdapter];
+  constructor(
+    private draperAdapter: DraperAdapter,
+    private feedAdapter: FeedAdapter,
+    private qsdAdapter: QSDAdapter,
+  ) {
+    this.adapters = [qsdAdapter, draperAdapter, feedAdapter];
```

Comment:
I assume draper should go above feed?  Though I don't know why anyone would use both.